Python/Product/PythonTools/PythonTools/Intellisense/CompletionAnalysis.cs

```diff
@@ -103,7 +113,7 @@ public class CompletionAnalysis {
 
         internal AnalysisEntry GetAnalysisEntry() {
             AnalysisEntry entry;
-            if (TextBuffer.TryGetAnalysisEntry(out entry) && entry != null) {
+            if (_view.TryGetAnalysisEntry(TextBuffer, _serviceProvider, out entry) && entry != null) {
```

Comment:
Does this mean analysis entries are more closely tied to views than buffers? That seems wrong...